cf/regrid/regrid.py

```diff
@@ -2400,8 +2564,30 @@ def create_esmpy_weights(
             row = None
             col = None
 
+    if esmpy_regrid_operator is not None:
+        # If we're returning the esmpy regrid operator, we need to
+        # create it by computing the weights.
+        compute_weights = True
+
     from_file = True
-    if compute_weights or esmpy_regrid_operator is not None:
+    if compute_weights:  # or esmpy_regrid_operator is not None:
```

Comment:
Is the comment here meant as a note, or is this logic commented-out as left over from development or similar?